RxJavaTemplate/buildSrc/src/main/java/Versions.kt

```diff
@@ -0,0 +1,56 @@
+object Versions {
+     const val BUILD_GRADLE_VERSION = "4.2.2"
+
+     const val ANDROID_VERSION_CODE = 9
+     const val ANDROID_VERSION_NAME = "3.1.0"
+     const val ANDROID_MIN_SDK_VERSION = 23
+     const val ANDROID_TARGET_SDK_VERSION = 28
```

Comment:
According to https://developer.android.com/distribute/best-practices/develop/target-sdk, could we change the ANDROID_TARGET_SDK_VERSION version to 30? And the ANDROID_COMPILE_SDK_VERSION as well

Reply:
@doannimble 👍 About the scope of the update, I'd like to keep all the current versions rather than an update on this PR, it might cause unexpected issues 😄 We would open a new PR to update the version.

Reply:
Okay, if it is our current development process, then we will need the new PR later.